src/transport/GasTransport.cpp

```diff
@@ -839,4 +844,90 @@ void GasTransport::getBinDiffusivityPolynomials(size_t i, size_t j, double* coef
     }
 }
 
+void GasTransport::getCollisionIntegralPolynomial(size_t i, size_t j, double* astar_coeffs, 
+                                                double* bstar_coeffs, double* cstar_coeffs) const
+{   
+    for (size_t k = 0; k < (m_mode == CK_Mode ? 6 : COLL_INT_POLY_DEGREE)+1; k++) {
```

Comment:
Missing white spaces ... should be `(m_mode == CK_Mode ? 6 : COLL_INT_POLY_DEGREE) + 1;`

Reply:
OK